xcodebuild/xcodebuild.go

```diff
@@ -56,23 +58,16 @@ func (b *xcodebuild) Version() (Version, error) {
 
 // Params ...
 type Params struct {
-	Action                    string
-	ProjectPath               string
-	Scheme                    string
-	DeviceDestination         string
-	CleanBuild                bool
-	DisableIndexWhileBuilding bool
-}
-
-// RunBuild ...
-func (b *xcodebuild) RunBuild(buildParams Params, outputTool string) (string, int, error) {
-	return b.runBuild(buildParams, outputTool)
+	Action      string
```

Comment:
NOTE: it would be great to get rid of this property. It stores `-workspace` or `-project`, based on the ProjectPath's extension. The problem is that xcodebuild's actions are totally different things, this is ProjectOrWorkspace option.

Reply:
So you suggest renaming it to `ProjectOrWorkspace`, right?